workers/unity/spatialos.UnityClient.worker.json

```diff
@@ -48,22 +48,21 @@
         ]
       },
       "macos": {
-          "command": "open",
-          "arguments": [
-            "./build/worker/UnityClient@Mac/UnityClient@Mac.app",
-            "--args",
-            "+assemblyName",
-            "local_assembly",
-            "+workerType",
-            "UnityClient",
-            "+infraServicesUrl",
-            "http://127.0.0.1:21000",
-            "+projectName",
-            "${IMPROBABLE_PROJECT_NAME}",
-            "-logfile",
-            "../../logs/external-default-unityclient.log"
-          ]
-        }
+        "command": "open",
```

Comment:
turns out that this file was always invalid json (one closing bracket too many), but none of our tools cared 😂

Reply:
🙈 💥 🤕